ssl/methods.c

```diff
@@ -11,209 +11,156 @@
 #include <openssl/objects.h>
 #include "ssl_locl.h"
 
-/*
+/*-
  * TLS/SSLv3 methods
  */
 
 IMPLEMENT_tls_meth_func(TLS_ANY_VERSION, 0, 0,
                         TLS_method,
                         ossl_statem_accept,
                         ossl_statem_connect, TLSv1_2_enc_data)
-
 #ifndef OPENSSL_NO_TLS1_2_METHOD
-IMPLEMENT_tls_meth_func(TLS1_2_VERSION, 0, SSL_OP_NO_TLSv1_2,
+    IMPLEMENT_tls_meth_func(TLS1_2_VERSION, 0, SSL_OP_NO_TLSv1_2,
```

Comment:
If you mean that #1414 doesn't reset it back, that's entirely correct.  What #1414 does is to mark it so it doesn't get auto-indented in the first place.